physical_ai_manager/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Build stage
+FROM node:20 AS build
```

Comment:
Node.js recommends using 22 or higher.

For more information, see https://nodejs.org/en/about/previous-releases

Reply:
@ola31 What do you think? Could you make a simple swap and test it out? I think changing it should be possible without any major issues.